docs/fdc3-compliance.md

```diff
@@ -70,6 +70,28 @@ FDC3 adopts the following experimental features policy:
 5. Experimental features are exempted from the normal versioning and deprecation policies that govern changes to FDC3. I.e. breaking changes may be made to experimental features between versions of the Standard without a major version release.
 6. The experimental designation may be removed from a feature in a minor version release (as this will be considered an additive change).
 
+## Conformance testing
+
+The FDC3 Standards include a set of [definitions for conformance tests](api/conformance/Conformance-Overview) that may be used to determine if a Desktop Agent API implementation conforms to a particular standard version, to help disambiguate complex parts of the FDC3 Standard and to enable test-driven development of a Desktop Agent implementation. The current set of tests focus on the Desktop Agent API and the interface to it. There are no tests currently defined for the Intents, Context, AppD or Bridging Parts of the FDC3 Standard. Hence, conformance for those parts of the Standard must be determined manually.
```

Comment:
There are tests for raising an intent (fdc3.raiseIntent), which is not the same thing as testing that the definiton of an intent conforms to the rules in the standard. Hopefully, that clarifies what I mean. That said, glad you've asked the question as it indicates that I need to revise this copy to avoid confusion!

Reply:
Got it! Yes, it would be helpful to revise the wording, Thank you!

Reply:
I've updated the copy and just dropped intents and context from it - I couldn't find a great way to word the statement that they are not APIs and don't get implemented so conformance testing is not needed (other than when reviewing to see if they follow guidelines).